Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs

```diff
@@ -216,8 +216,12 @@ private Transform MakeOffsetTransform(Transform parentTransform)
             transformWithOffset.transform.localPosition = AdditionalOffset;
             transformWithOffset.transform.localRotation = Quaternion.Euler(AdditionalRotation);
             transformWithOffset.name = string.Format("{0} on {1} with offset {2}, {3}", gameObject.name, TrackedObjectToReference.ToString(), AdditionalOffset, AdditionalRotation);
-            // In order to account for the reversed normals due to the glTF coordinate system change, we need to provide rotated attachment points.
-            transformWithOffset.transform.Rotate(0, 180, 0);
+
+            if (TrackedObjectToReference != TrackedObjectToReferenceEnum.Head)
```

Comment:
Will things like puck trackers and other accessories need to be reversed too?

Reply:
Likely not, but those types also likely won't go into the legacy Toolkit. We'll need to address this a little more elegantly in vNext, since the legacy Toolkit assumes that the only platform controller models we'll receive are from the Windows MR APIs.

I can rework it here first if you'd prefer, but I don't think it's a scenario we'll encounter.

Reply:
Ah, I missed which branch this was going into!